FWCore/Framework/interface/InputSource.h

```diff
@@ -424,7 +427,7 @@ namespace edm {
 
   private:
 
-    std::shared_ptr<ActivityRegistry> actReg_;
+    mutable std::shared_ptr<ActivityRegistry> actReg_;
```

Comment:
@Dr15Jones Very likely yes. Also, I asked you if the ActivityRegistry could be mutable, and you said yes, because it is thread safe. I know for sure that some of the actReg_ data members needed to be modified by a const function in one or more classes, but I am not 100% sure that this is true in each and every class that holds the shared pointer.